assemblies/pad-ce/src/main/resources/lib/log4j2.xml

```diff
@@ -1,21 +1,65 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<Configuration status="WARN">
-  <Appenders>
-    <RollingFile name="FILE" fileName="aggdesigner.log" filePattern="aggdesigner.log.%i" append="true">
-      <PatternLayout pattern="%d %-5p [%c] %m%n"/>
-      <SizeBasedTriggeringPolicy size="500KB" />
-      <DefaultRolloverStrategy max="1" fileIndex="min"/>
-    </RollingFile>
-    <Console name="CONSOLE">
-      <PatternLayout>
-        <Pattern>%d{ABSOLUTE} %-5p [%c{1}] %m%n</Pattern>
-      </PatternLayout>
-    </Console>
-  </Appenders>
-  <Loggers>
-    <Root level="info">
-      <AppenderRef ref="FILE"/>
-      <AppenderRef ref="CONSOLE"/>
-    </Root>
-  </Loggers>
+<!-- ===================================================================== -->
+<!--                                                                       -->
+<!--  Log4j2 Configuration                                                  -->
+<!--                                                                       -->
+<!-- ===================================================================== -->
+
+<!--
+   | For more configuration information and examples see the Apache Log4j
+   | website: https://logging.apache.org/log4j/2.x/manual/configuration.html
+ -->
+
+<Configuration name="AggDesigner" status="INFO" dest="out">
+
+    <Appenders>
+
+        <!-- ================================= -->
+        <!-- Preserve messages in a local file -->
+        <!-- ================================= -->
+
+        <!-- A time/date based rolling appender -->
+        <RollingFile name="FILE" fileName="aggdesigner.log" filePattern="aggdesigner.log.%d{yyyy-MM-dd}" append="false" bufferedIO="true" immediateFlush="true">
```

Comment:
@lucboudreau 
The original log4j.xml uses a RollingFileAppender with MaxFileSize=500KB and MaxBackupIndex=1, so this generates files like:  aggdesigner.log and aggdesigner.log.1

The corresponding log4j2.xml to get the same behavior would look like the MONDRIAN appender here:
https://github.com/pentaho/pentaho-platform/blob/log4j2/assemblies/pentaho-war/src/main/webapp/WEB-INF/classes/log4j2.xml#L59

Reply:
@bennychow - Thanks for the review. Fix is here.
https://github.com/pentaho/pentaho-aggdesigner/pull/144